src/views/AddLead/utils.js

```diff
@@ -51,6 +52,14 @@ export function fillExtraInfo(inputValues, currentUserId, leadOptions = {}) {
     ) {
         values.confidentiality = leadOptions.confidentiality[0].key;
     }
+    if (
+        !values.priority
```

Comment:
why not use `isDefined` here? it's unsafe to use this syntax to check for undefined vaues for number.